rules/manager.go

```diff
@@ -281,7 +282,16 @@ func (m *Manager) ApplyConfig(conf *config.Config) {
 	copy(rulesSnapshot, m.rules)
 	m.rules = m.rules[:0]
 
-	if err := m.loadRuleFiles(conf.RuleFiles...); err != nil {
+	var files []string
+	for _, pat := range conf.RuleFiles {
+		fs, err := filepath.Glob(pat)
```

Comment:
I thought we had settled on the fixed `<directory>/*.rules` approach. But now I see that file_sd_configs is actually following the glob approach (in contrast to the textfile module of the node exporter). Also, we do not seem to have a clear convention to end rules files on `.rules`.

So your approach seems sane here. I updated the issue.